keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Priority and Fairness for API Server Requests
+authors:
+  - "@MikeSpreitzer"
+  - "@yue9944882"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - wg-multitenancy
```

Comment:
not actually a sig, although I agree it makes sense to list them

Reply:
Yes.  I think there is an issue here for the KEP template.